name.abuchen.portfolio/src/name/abuchen/portfolio/json/JSecurity.java

```diff
@@ -9,7 +9,7 @@
     private String name;
     private String isin;
     private String wkn;
-    private String ticker;
+    private String tickerSymbol;
     private String currency;
```

Comment:
This is a breaking change in the JSON that the user can download the view "All transactions" -> Toolbar: Export -> All transactions (JSON). Why? The json is created using the property names of the Java class. 

To be honest, I do not know how many users are using this export. So maybe we can get away with the change here